browser/app/js/browser/Login.js

```diff
@@ -148,7 +148,8 @@ export class Login extends React.Component {
                   <OpenIDLoginButton
                     className="btn openid-btn"
                     clientId={this.state.clientId}
-                    authorizationEndpoint={this.state.discoveryDoc.authorization_endpoint}
+                    authEp={this.state.discoveryDoc.authorization_endpoint}
+                    authScopes={this.state.discoveryDoc.supported_scopes}
```

Comment:
It should be `scopes_supported`.
Example configuration is here https://mk-dev1.auth0.com/.well-known/openid-configuration